packages/legend-manual-tests/src/__tests__/query-execution/TEMPORARY__QueryBuilderExecution.test.tsx

```diff
@@ -178,8 +178,16 @@ test(integrationTest('test query execution with parameters'), async () => {
   const queryBuilderExploreTree = await waitFor(() =>
     renderResult.getByTestId(QUERY_BUILDER_TEST_ID.QUERY_BUILDER_EXPLORER),
   );
-  const propertyNode = getByText(queryBuilderExploreTree, 'First Name');
-  await waitFor(() => fireEvent.contextMenu(propertyNode));
+  const FirstNamePropertyNode = getByText(
```

Comment:
Can we also enhance the integration test 'Query builder result state' here: https://github.com/finos/legend-studio/pull/2622

That test is easier to see as it just loads the execution result then you see the data in tree.